### PR TITLE
Update GPL text and FSF address

### PR DESCRIPTION
**Pull request by jamesjer**
- Update the FSF's address; they moved from 59 Temple Place in 2005.
- Update the GPL text to the contents of http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.

### CODE REVIEW

File: lib/matobj2.gd [hunk 1]
```diff
@@ -807,7 +807,7 @@ DeclareOperation( "{}", [IsFlatMatrix,IsList] );
 ##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ##  GNU General Public License for more details.
 ##
-##  You should have received a copy of the GNU General Public License
-##  along with this program; if not, write to the Free Software
-##  Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+##  You should have received a copy of the GNU General Public License along
```
Comment (alex-konovalov) [inline lib/matobj2.gd]: This is the only library file with these details - perhaps we may remove them from here for consistency.

**Comment by markuspf** [comment]
Further to @alex-konovalov s comment one could actually remove redundant copies of the GPL and just have one of it in the source tree.

**Comment by ChrisJefferson** [comment]
This patch looks OK to me, just text changes.

**Comment by alex-konovalov** [comment]
I am closing this PR since it is incorporated into PR #888.

**Pull request closed without merging**